middle-pgsql.cpp

```diff
@@ -1063,6 +1073,7 @@ void middle_pgsql_t::start(const options_t *out_options_)
         pgsql_exec(sql_conn, PGRES_COMMAND_OK, "SET synchronous_commit TO off;");
 
         pgsql_exec(sql_conn, PGRES_COMMAND_OK, "SET client_min_messages = WARNING");
+        pgsql_exec(sql_conn, PGRES_COMMAND_OK, "CREATE SCHEMA IF NOT EXISTS %s", schema.c_str());
```

Comment:
We still support 9.1 which doesn't support the `IF NOT EXISTS` parameter. I recommend dropping this as we already rely on the database having appropriate extensions and this is similar.

Reply:
The README file still lists 9.1 as minimum version, although it isn't supported anymore since more than 2 years now. Should this updated?

https://www.postgresql.org/support/versioning/